src/test/java/nextstep/subway/domain/LineTest.java

```diff
@@ -0,0 +1,42 @@
+package nextstep.subway.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class LineTest {
+    private Line 역2개_2호선;
+    private Line 역3개_2호선;
+    private Station 서초역;
+    private Station 강남역;
+    private Station 교대역;
+
+    @BeforeEach
+    void setUp() {
+        서초역 = Station.from("서초역");
+        교대역 = Station.from("교대역");
+        강남역 = Station.from("강남역");
+        역2개_2호선 = Line.builder().upStation(서초역).downStation(강남역).distance(10).build();
+        역3개_2호선 = Line.builder().upStation(서초역).downStation(강남역).distance(10).build();
+        역3개_2호선.addSection(Section.from(서초역, 교대역, Distance.from(5)));
+    }
+
+    @Test
+    void 노선에_할당된_역을_조회할_수_있다() {
+        assertThat(역2개_2호선.findAssignedStations()).containsExactly(서초역, 강남역);
+    }
+
+    @Test
+    void 구간을_추가하고_노선에_할당된_역을_조회시_추가한_역이_조회된다() {
```

Comment:
> Line 도메인에 대한 테스트 코드를 추가했습니다.[42df20b](https://github.com/next-step/atdd-subway-admin/pull/952/commits/42df20b9428521bfc11e6a2b905acff5e5f3f3d6)
https://blog.kingbbode.com/m/52?category=737337 를 읽어보았는데, 테스트에서는 테스트의 의도를 명확히 나타낼 수 있는게 좋을 것 같아, 의도를 쉽게 파악할 수 있도록 적어보려고 노력했는데 잘 작성한건지는 모르겠습니다. 😢

이 부분에 좀 더 첨언을 드리면 `@DisplayName`을 활용한다면 의도를 파악하기 더 쉬울 것 같아요 😄 
```suggestion
    @DisplayName("구간을 추가하고 노션 조회시 추가한 역 조회를 성공한다.")
    @Test
    void addSections_success() {
```

Reply:
@ohtaeg 
제안해주신 것 처럼 `@DisplayName`를 활용하도록 변경했습니다! 😄